core/base/document.php

```diff
@@ -86,7 +86,7 @@ public static function get_editor_panel_config() {
 			'elements_categories' => static::get_editor_panel_categories(),
 			'messages' => [
 				/* translators: %s: the document title. */
-				'publish_notification' => sprintf( __( 'Hurray! Your %s is live.', 'elementor' ), self::get_title() ),
+				'publish_notification' => sprintf( __( 'Hurray! Your %s is live.', 'elementor' ), static::get_title() ),
```

Comment:
Make `get_editor_panel_config` not `static` & use `$this->get_post_type_title()`.
Then override `get_post_type_title` in the `Library_Document` and return `static::get_title()`.

Reply:
done